security/index.adoc

```diff
@@ -0,0 +1,88 @@
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-intro-about-this-guide]]
+== About This Guide
+
+This guide provides a high-level walkthrough of the container security measures
+available in {product-title}, including solutions for the host layer, the
+container and orchestration layer, and the build and application layer. After
+reading this guide, you should be able to identify:
+
+- Why container security is important and how it compares with existing security standards.
+- Which container security measures are provided by the host (RHEL) layer and which are provided uniquely from {product-title}.
+- How to evaluate your container content and sources for vulnerabilities.
+- How to design your build and deployment process to proactively check container content.
+- How to control access to containers via authentication and authorization.
+- How networking and attached storage are secured in {product-title}.
+- Containerized solutions for API management and SSO.
+
+[[security-intro-what-are-containers]]
+== What Are Containers?
+
+Containers have broad appeal for the ability they provide to easily package an
+application and all its dependencies into a single image that can be promoted
+from development, to test, to production, without change.
+
+Containers make it easy to ensure consistency across environments and multiple
+deployment targets: physical servers, virtual machines (VMs), and private or
+public cloud. All of this helps teams to develop and manage the applications
+that drive business value with greater agility.
+
+Therefore, depending on whether you are asking a system administrator or an
+application developer, the benefits of containers may sound slightly different:
+
+// image::whatarecontainers.png["What Are Containers?", align="center"]
+
+[options="header",frame="all",grid="none"]
+|===
+|INFRASTRUCTURE |APPLICATIONS
+
+|Sandboxed application processes on a shared Linux OS kernel
+|Package my application and all of its dependencies
+
+|Simpler, lighter, and denser than virtual machines
+|Deploy to any environment in seconds and enable CI/CD
+
+|Portable across different environments
+|Easily access and share containerized components
+|===
+
+[discrete]
+[[security-intro-further-reading-1]]
+==== Further Reading
+
+- _{product-title} Architecture_:  xref:{sourcedir}architecture/core_concepts/containers_and_images.adoc#architecture-core-concepts-containers-and-images[Core Concepts -> Containers and Images]
+
+[[security-intro-enterprise]]
+== Securing Containers for Enterprise Use
+
+Enterprise use of containers requires strong security. In many ways, securing
+containers is a lot like securing any running process. You need to think about
+security throughout the layers of the solution stack before you deploy and run
+your containers. You also need to think about security throughout the
+application and container life cycle.
```

Comment:
Linking to RHEL security guides here, and showing / proving that the same security guides for RHEL apply here may help better make this clear to customers.